src/_pytest/runner.py

```diff
@@ -64,35 +65,43 @@ def pytest_addoption(parser: Parser) -> None:
 
 
 def pytest_terminal_summary(terminalreporter: "TerminalReporter") -> None:
-    durations = terminalreporter.config.option.durations
-    durations_min = terminalreporter.config.option.durations_min
-    verbose = terminalreporter.config.getvalue("verbose")
+    durations = terminalreporter.config.getoption("durations")
     if durations is None:
         return
-    tr = terminalreporter
-    dlist = []
-    for replist in tr.stats.values():
-        for rep in replist:
-            if hasattr(rep, "duration"):
-                dlist.append(rep)
-    if not dlist:
+
+    durations_min = terminalreporter.config.getoption("durations_min")
+    verbose = terminalreporter.config.getvalue("verbose")
+    reporter = terminalreporter
+    durations_list = []
+    for report_list in reporter.stats.values():
+        for report in report_list:
+            if hasattr(report, "duration"):
+                durations_list.append(report)
+    if not durations_list:
         return
-    dlist.sort(key=lambda x: x.duration, reverse=True)  # type: ignore[no-any-return]
+
+    durations_list.sort(key=attrgetter("duration"), reverse=True)
```

Comment:
I prefer the lambda only because `attrgetter` gets the field name as a string which means it cannot be typed.

Reply:
(I mean type *checked*)